packages/prisma-faker/src/seed.ts

```diff
@@ -1331,18 +1332,39 @@ export function seed<
             // TODO:
             let seed
             try {
+              const { id, ...dataWithoutId } = f.data
               seed = await photon[f.mapping.findMany]['create']({
-                data: f.data,
+                data: dataWithoutId,
```

Comment:
To my understanding, that should break relations.

Reply:
This is a temporary workaround as bring your own id feature is broken in engine at the moment. The example that I use for now does not have any relations on purpose.